tests/test.hs

```diff
@@ -129,7 +131,8 @@ prop_alt_cont = once $
 
 prop_alt_help :: Property
 prop_alt_help = once $
-  let p = p1 <|> p2 <|> p3
+  let p :: Parser (Maybe (Either String String))
```

Comment:
FWIW: this is breaking change, as the need to alter tests highlights, so would need a major version bump.